test/lightning_web/channels/attempt_channel_test.exs

```diff
@@ -240,9 +240,15 @@ defmodule LightningWeb.AttemptChannelTest do
 
   describe "cast credential types" do
     defp set_postgresql_credential(_context) do
-      credential =
-        insert(:credential,
+      user = insert(:user)
+
+      # TODO - roger, if we want to check that the casting works could we use it
+      # here and verify that the output is valid for the worker? the below test
```

Comment:
just a negative scope test, to show that we are not casting on the reply as @stuartc requested